lib/util/http/cache/abstract-http-cache-provider.ts

```diff
@@ -66,11 +66,11 @@ export abstract class AbstractHttpCacheProvider implements HttpCacheProvider {
         timestamp,
       });
 
-      /* v8 ignore next 4 -- should never happen */
+      /* v8 ignore start: should never happen */
```

Comment:
why this change 🤔

Reply:
I don't like the line count approach

Reply:
I dislike both approaches, but start/stop seems better of the two